openc3-cosmos-init/plugins/packages/openc3-vue-common/src/tools/scriptrunner/RunningScripts.vue

```diff
@@ -210,7 +210,7 @@ export default {
     return {
       api: null,
       timeZone: 'local',
-      activeTab: 'running',
+      activeTab: 'completed',
```

Comment:
Shouldn't showing the running scripts be the default?

Reply:
I think most users want to see completed scripts because they're not migrating away from the current running script.